sdks/java/testing/test-utils/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.vendored_guava_26_0_jre
   compile library.java.google_cloud_bigquery
+  // google_cloud_bigquery is not declaring google_http_client_jackson dependency correctly
+  compile library.java.google_http_client_jackson
```

Comment:
Without this depenency, `HadoopFormatIOIT` fails: https://github.com/apache/beam/pull/10327#issuecomment-563311214